app/models/notification.rb

```diff
@@ -75,7 +75,9 @@ def self.ensure_consistency!
     DB.exec(<<~SQL)
       DELETE
         FROM notifications n
-       WHERE high_priority AND notification_type <> #{types[:chat_mention].to_i}
+       WHERE high_priority
+         AND notification_type <> #{types[:chat_mention].to_i}
+         AND notification_type <> #{types[:chat_message].to_i}
```

Comment:
did you mean to double up these lines?

Reply:
One is `chat_mention`, one is `chat_message`. Obviously the first one is for mentioning, but the second is for when you're notification preferences are to get a notification on every new chat message in a channel.

Reply:
Wow, I'm dumb! I swear I re-read that 10 times to be sure.